asserts/snapasserts/validation_sets.go

```diff
@@ -584,3 +585,37 @@ func (v *ValidationSets) CheckPresenceInvalid(snapRef naming.SnapRef) ([]string,
 	sort.Strings(keys)
 	return keys, nil
 }
+
+// ParseValidationSet parses a validation set string (account/name or account/name=sequence)
+// and returns its individual components, or an error.
+func ParseValidationSet(arg string) (account, name string, seq int, err error) {
+	errPrefix := func() string {
+		return fmt.Sprintf("cannot parse validation set %q", arg)
+	}
+	parts := strings.Split(arg, "=")
+	if len(parts) > 2 {
+		return "", "", 0, fmt.Errorf("%s: expected account/name=seq", errPrefix())
```

Comment:
Since `arg` never changes, we could just compute the prefix once and use it without rebuilding it at every call
```suggestion
	errPrefix := fmt.Sprintf("cannot parse validation set %q", arg)
	parts := strings.Split(arg, "=")
	if len(parts) > 2 {
		return "", "", 0, fmt.Errorf("%s: expected account/name=seq", errPrefix)
```
And then replace it in the other usages of `errPrefix()` below

Reply:
I guess the reason for not doing it is that is to save some cycles in the happy case, where this prefix does not need to be computed at all.